### ISSUE TITLE
Make use of nubis-builder generated AMIs.json

### ISSUE THREAD

**Issue by gozer** [nubisproject/nubis-ci#407]
Once nubisproject/nubis-builder#208 lands, we don't need to parse Packer's output anymore.

This way, we can remove the machine-output option and get coloring

And we get rid of some yucky parsing, and could replace it instead with something like:

```bash
$> cat nubis/builder/artifacts/AMIs.json | jq -r ".builds[] | select (.packer_run_uuid | contains(\"$(jq -r .last_run_uuid < nubis/builder/artifacts/AMIs.json)\")) .artifact_id"
us-east-1:ami-XXXXXXXX,us-west-2:ami-YYYYYYYY
```

**Comment by gozer** [comment]
Actually better use of jq would be 👍 
```bash
$> jq -r '.last_run_uuid as $uuid | .builds[] | select (.packer_run_uuid == $uuid) .artifact_id' nubis/builder/artifacts/AMIs.json
```

**Comment by gozer** [comment]
Closed by #429
